app/controllers/AlertController.java

```diff
@@ -61,6 +70,53 @@ public AlertController(final Configuration configuration, final AlertRepository
     }
 
     /**
+     * Adds or updates an alert in the alert repository.
+     * If a valid UUID is passed, it will update the corresponding alert if it exists or create an alert with this id.
```

Comment:
Hrm. Should we not distinguish this based on PUT (update) versus POST (create)? There is likely common logic between these two controller methods but the entry points seem separate. @BrandonArp thoughts?